hyperthreadingdisabled/vars/main.yml

```diff
@@ -0,0 +1 @@
+#  internet: true
```

Comment:
is this right, adding a file that is in .gitignore?

Reply:
Good catch,